tests/rules/metavar_call_case_insensitive.yaml

```diff
@@ -0,0 +1,7 @@
+rules:
+- id: test
+  message: Php function references are case insensitive.
+  languages: [php]
+  severity: WARNING
+  patterns:
+  - pattern: $FUN(...) == $FUN(...)
```

Comment:
Replace $FUN(...) by $X here, and see if it still works.

Reply:
Yeah, this doesn't work. Left a todo here and in Matching_generic.ml . We will get it in the next diff.